Frontend/DinoTracker/app/styles/app.css

```diff
@@ -20,6 +20,23 @@
     font-size: 20px;
 }
 
+.input-field {

```

Comment:
Good job recognizing that these will be applied globally and not just in that routes css file.